pdk.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'tty-spinner', '~> 0.4'
   spec.add_runtime_dependency 'tty-prompt', '~> 0.12'
   spec.add_runtime_dependency 'json_pure', '~> 2.1.0'
+  spec.add_runtime_dependency 'tty-which', '~> 0.3.0'
```

Comment:
Addition of this dependency will require some work on the packaging side. Going to hold this for 0.5.0 so we can have time for packaging work.